doc: add descripions in chart/value.yaml (#7515)
longhorn/longhorn: chart/README.md

| annotations | `{}` | Annotations to add to the Longhorn Manager DaemonSet Pods. Optional. |
| enableGoCoverDir | `false` | Enable this to allow Longhorn to generate code coverage profiles |
| enablePSP | `false` | For Kubernetes < v1.25, if your cluster enables Pod Security Policy admission controller, set this to `true` to ship a built-in longhorn-psp PodSecurityPolicy which allow privileged Longhorn pods to start |

> | helmPreUpgradeCheckerJob.enabled | `true` | Enable this to allow Longhorn to do pre-upgrade checks. Disable this to support ArgoCD. |

Reviewer: ```suggestion
| helmPreUpgradeCheckerJob.enabled | `true` | Setting that allows Longhorn to perform pre-upgrade checks. Disable this setting when installing Longhorn using Argo CD. |
```

Author: Modified. Thanks.

> | metrics.serviceMonitor.enabled | `false` | Enable this to create the ServiceMonitor |

Reviewer: ```suggestion
| metrics.serviceMonitor.enabled | `false` | Setting that allows the creation of ServiceMonitor. |
```

Author: Modified. Thanks.

| defaultSettings.storageReservedPercentageForDefaultDisk | The reserved percentage specifies the percentage of disk space that will not be allocated to the default disk on each new Longhorn node. |
| defaultSettings.supportBundleFailedHistoryLimit | This setting specifies how many failed support bundles can exist in the cluster. Set this value to **0** to have Longhorn automatically purge all failed support bundles. |
| defaultSettings.systemManagedComponentsNodeSelector | nodeSelector for Longhorn system-managed components |
| defaultSettings.systemManagedPodsImagePullPolicy | This setting defines the Image Pull Policy of Longhorn system managed pod. E.g. instance manager, engine image, CSI driver, etc. The new Image Pull Policy will only apply after the system-managed pods restart. |
| defaultSettings.taintToleration | taintToleration for Longhorn system-managed components |
| defaultSettings.upgradeChecker | Upgrade Checker will check for a new Longhorn version periodically. When there is a new version available, a notification will appear in the UI. By default, true. |

> | defaultSettings.v1DataEngine | Setting that allows you to enable the V1 Data Engine. By default, true. |

Reviewer: ```suggestion
| defaultSettings.v1DataEngine | Setting that allows you to enable the V1 Data Engine. This setting is enabled by default. |
```

Author: Modified. Thanks.

> | defaultSettings.v2DataEngine | This allows users to activate v2 data engine based on SPDK. By default, false. Currently, it is in the preview phase and should not be utilized in a production environment. |

Reviewer: ```suggestion
| defaultSettings.v2DataEngine | Setting that allows you to enable the V2 Data Engine, which is based on the Storage Performance Development Kit (SPDK). The V2 Data Engine is a preview feature and should not be used in production environments. |
```

Author: Modified. Thanks.
